### ISSUE TITLE
Add "autocomplete" attributes to forms across site

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1374]
Newsletter form, downloads form
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

**Acceptance Criteria:**
- [ ] The "Name" and "Email" field should have the "autocomplete" tag everywhere it appears (newsletter)
- [ ] "Email" field on the lesson download
    - I am pretty sure the autocomplete function will only fill the field in if the user wants it to be filled in. If that is not the case, and any interaction with the form autocompletes it, we probably don't want it here as the email is optional
